src/lib/Interceptor/index.test.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint camelcase: ["error", {allow: ["snake_case_key"]}] */
```

Comment:
Do we need this? 🤔

Reply:
Yes, we need just only this file because of mocking the `Axios` response data.